recipe/conda_build_config.yaml

```
@@ -2,3 +2,5 @@ c_compiler:
   - vs2022                     # [win]
 cxx_compiler:
   - vs2022                     # [win]
+c_stdlib_version:
+  - "2.17"
```

As this only applies to Linux, would suggest this change

```suggestion
c_stdlib_version:              # [linux]
  - "2.17"                     # [linux]
```